templates/includes/base.html

```diff
@@ -4,45 +4,26 @@
         <title>Twitter</title>
         <meta charset="utf-8">
         <meta name="viewport" content="width=device-width, initial-scale=1">
-
         {% block specific_meta %}
-
         {% endblock %}
-
-
-
     </head>
     <body>
-
         <div class="container">
             <div class="row">
-
-                <div class="col-md-3">
-                    <div class="left_sidebar">
-
-                    </div>
-                </div>
+                <div class="col-md-3"></div>
                 <div class="col-md-6">
```

Comment:
Why are you going with Bootstrap class names convention when we are not using Bootstrap

Reply:
These are not your styles. Just copy paste from Bootsrap. 

Thats why you. have added empty `<div class="col-md-3"></div>` to centre align and give margin on both sides.


```
.container{
    -bs-gutter-x: 1.5rem;
    --bs-gutter-y: 0;
    width: 100%;
    padding-right: calc(var(--bs-gutter-x)* .5);
    padding-left: calc(var(--bs-gutter-x)* .5);
    margin-right: auto;
    margin-left: auto;
}
```

1. This is first few lines from main.css. Copy pasted from Bootstrap css.
2. `bs` -> bootstrap
3. remember your old commit `remove bs` implies remove bootstrap.